_scss/_buttons.scss

```diff
@@ -38,7 +38,7 @@ a.button.outline-btn.min-hgt {
     border: 1px solid rgba(0, 0, 0, 0.1);
     text-shadow: 0 1px rgba(0, 0, 0, 0.2);
 }
-
+/*
```

Comment:
FWIW; I kept this one in for now, in case we need another button style again (we could remove if not)